OpenRA.Game/Effects/IEffect.cs

```diff
@@ -20,5 +20,8 @@ public interface IEffect
 		IEnumerable<IRenderable> Render(WorldRenderer r);
 	}
 
+	// Identifier interface for effects that are added to ScreenMap
+	public interface ISpatiallyPartitionable { }
```

Comment:
How about an `ISpatiallyPartitionedEffect : IEffect { }` instead of requiring both?

Reply:
Originally I was doing that, but that would then later force us to migrate all projectiles in one go. Might make sense anyway, just explaining my train of thought behind using a separate interface.

Reply:
Oh right, I didn't think about the `IProjectile : IEffect`.